telegram/_user.py

```diff
@@ -988,6 +1001,11 @@ async def send_video(
         caption_entities: Union[List["MessageEntity"], Tuple["MessageEntity", ...]] = None,
         filename: str = None,
         protect_content: ODVInput[bool] = DEFAULT_NONE,
+        *,
```

Comment:
same here, `api_kwargs` for `User.send_video` should be moved under the asterisk.